error getting FileResource from DatabaseDesign

- Domino server version -12.0.1 F! 1
- ODA version - 12.0.3-SNAPSHOT

The `getFileResource` from `DatabaseDesign ` throws exception while decoding base64.
```
    DatabaseDesign design = db.getDesign();
    FileResource file = design.getFileResource("VideoFrame.html");
```

Exception:

```
Caused by: java.lang.IllegalArgumentException: Illegal base64 character a
	at java.util.Base64$Decoder.decode0(Base64.java:714)
	at java.util.Base64$Decoder.decode(Base64.java:526)
	at java.util.Base64$Decoder.decode(Base64.java:549)
	at org.openntf.domino.design.impl.AbstractDesignFileResource.getFileData(AbstractDesignFileResource.java:94)

```

--- a/domino/core/src/main/java/org/openntf/domino/design/impl/AbstractDesignFileResource.java
+++ b/domino/core/src/main/java/org/openntf/domino/design/impl/AbstractDesignFileResource.java
@@ -91,7 +91,7 @@ public abstract class AbstractDesignFileResource extends AbstractDesignBaseNamed
 		switch (getDxlFormat(true)) {
 		case DXL:
 			String rawData = getDxl().selectSingleNode("//filedata").getText(); //$NON-NLS-1$
-			return Base64.getDecoder().decode(rawData);
+			return Base64.getMimeDecoder().decode(rawData);
 		default:
 			return getFileDataRaw(DEFAULT_FILEDATA_FIELD);
 
@@ -111,7 +111,7 @@ public abstract class AbstractDesignFileResource extends AbstractDesignBaseNamed
 					"//item[@name='" + XMLDocument.escapeXPathValue(itemName) + "']/rawitemdata")) { //$NON-NLS-1$ //$NON-NLS-2$
 
 				String rawData = rawitemdata.getText();
-				byte[] thisData = Base64.getDecoder().decode(rawData);
+				byte[] thisData = Base64.getMimeDecoder().decode(rawData);
 				byteStream.write(thisData);
 			}
 

--- a/domino/core/src/main/java/org/openntf/domino/design/impl/ImageResource.java
+++ b/domino/core/src/main/java/org/openntf/domino/design/impl/ImageResource.java
@@ -60,7 +60,7 @@ public final class ImageResource extends AbstractDesignFileResource implements o
 		switch (getDxlFormat(true)) {
 		case DXL:
 			String rawData = getDxl().selectSingleNode("//jpeg|//gif|//png").getText(); //$NON-NLS-1$
-			return Base64.getDecoder().decode(rawData);
+			return Base64.getMimeDecoder().decode(rawData);
 		default:
 			return getFileDataRaw("$ImageData"); //$NON-NLS-1$
 

